### PR TITLE
Updated edx-rest-api-client dependency.

### PR DESCRIPTION
**Pull request by dsjen**
@dan-f -- this is the proposed solution for [AN-6686](https://openedx.atlassian.net/browse/AN-6686).  Please see comments in the ticket.

~~I'm still waiting on confirmation on the exact version of slumber that is installed on the buggy AMI and won't merge this until then.~~  Confirmed!

### CODE REVIEW

File: analytics_dashboard/core/tests/test_utils.py [hunk 1]
```diff
@@ -55,11 +55,6 @@ class CourseStructureApiClientTests(TestCase):
     """
     Tests the CourseStructureApiClient wrapper class.
     """
-    @data((None, None), ('http://example.com/', None), (None, 'arbitrary_access_token'))
-    @unpack
-    def test_required_args(self, url, access_token):
```
Comment (dsjen) [inline analytics_dashboard/core/tests/test_utils.py:60]: FYI, the library no longer throws this error.  See: https://github.com/edx/edx-rest-api-client/commit/e9d763fe87148029dd4d8dd0de688959d5b2e5ab#diff-de9ec2e2333d1924c0ef078c95c45cdaL29

File: requirements/base.txt [hunk 2]
```diff
@@ -9,7 +9,7 @@ django-libsass==0.2			# BSD
 django-model-utils==1.5.0 	# BSD
 django-soapbox==1.1         # BSD
 django-waffle==0.10			# BSD
-edx-rest-api-client==1.2.1              # Apache
+edx-rest-api-client==1.5.0  # Apache
```
Comment (dan-f) [inline requirements/base.txt]: Can we use a version specifier that allows bugfix releases?

**Comment by dan-f** [comment]
:+1:

File: requirements/base.txt (hunk 2 above)
Comment (dsjen) [inline requirements/base.txt]: Thanks!  Done.

**Pull request merged**